doc/sphinx/source/esmvalcore/recipe.inc

```diff
@@ -24,12 +24,38 @@ The documentation section includes:
 - A list of scientific references
 - the project or projects associated with the recipe.
 
+For example, please see the documentation section from the recipe:
+recipe_ocean_amoc.yml.
+
+.. code-block:: yaml
+
+    documentation:
+      description: |
+        Recipe to produce time series figures of the derived variable, the
+        Atlantic meriodinal overturning circulation (AMOC).
+        This recipe also produces transect figures of the stream functions for
+        the years 2001-2004.
+
+      authors:
+        - demo_le
+
+      maintainer:
+        - demo_le
+
+      references:
+        - demora2018gmd
+
+      projects:
+        - ukesm
+
 Note that the author, project, and references will need to be included in the
 config-references.yml file. The author name uses the format:
-last name (clipped after 4 letters) underscore first name (clipped to two letters)
-and all letters are lowercase. 
-For instance, Mickey Mouse would be: mous_mi.
-This is unlikely to be the same as the github user name.
+last name (clipped after 4 letters) underscore first name (clipped to two
```

Comment:
Actually, the author name does not have to be in this format, it can be anything the user likes. This is a leftover from version 1.

Reply:
We might as well maintain the same standard, right?

Reply:
I'm not sure, it looks a bit user unfriendly, why this very strict description of what a user name should look like if it's not needed at all?

Reply:
We can introduce a different "rule" (like `name_surname`), but I think it's good to keep this uniform.

Reply:
That looks better already

Reply:
I don't think that this PR is the place to introduce a sweeping change which would affect every recipe! Do you want me to remove this section or change it in any way?

Reply:
Just write down the new scheme, so new authors will use that and then we can gradually replace the old names

Reply:
Not to be annoying, but I'd prefer to keep the last name first ie `surname_name`. Otherwise we'd have to re-order the entire list of authors in `config-developpers.yml`!

Reply:
I propose changing this paragraph to:

```
Note that the author, project, and references will need to be included in the
``config-references.yml`` file. The author name uses the format:
`surname_name`. For instance, Mickey Mouse would be: mouse_mickey.
Also note that this username is unlikely to be the same as the github
user name.
```

Reply:
> Not to be annoying, but I'd prefer to keep the last name first ie surname_name. Otherwise we'd have to re-order the entire list of authors in config-developpers.yml!

Yes, sure, good point!

Reply:
`author` --> `author(s)`
`project` --> `project(s)`

Reply:
Okay, fixed!